test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Html.Tests.ps1

```diff
@@ -168,5 +168,21 @@ After the object
         $returnString = $customObject | ConvertTo-HTML -Transitional | Select-Object -First 1
         $returnString | Should -Be '<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">'
     }
+
+    It "Test ConvertTo-HTML calculated property supports 'name' key as alias of 'label'" {
+        $returnString = ($customObject | ConvertTo-Html @{ name = 'AgeRenamed'; e = 'Age'}) -join $newLine
+        $returnString | Should -Match 'AgeRenamed'
+    }
+
+    It "Test ConvertTo-HTML calculated property supports integer 'width' entry" {
```

Comment:
Only integer?

Reply:
@iSazonov: Good question: `int` is the type of the `Alignment` entry for _Format-Table_, and I simply amended that class to also be able to use it with ConvertTo-Html, with a string-typed width entry.

Since we're talking about UIs and column widths, my guess is that `[int]` is enough, but I'm not wedded to that, and if someone knows that a wider type is called for, I'm all ears.